### ISSUE TITLE
completion errors out when opening a bracket in .adb (Ada) files

### ISSUE THREAD

**Issue by Orest58008** [echasnovski/mini.nvim#859]
### Contributing guidelines

- [X] I have read [CONTRIBUTING.md](https://github.com/echasnovski/mini.nvim/blob/main/CONTRIBUTING.md)
- [X] I have read [CODE_OF_CONDUCT.md](https://github.com/echasnovski/mini.nvim/blob/main/CODE_OF_CONDUCT.md)
- [X] I have updated 'mini.nvim' to latest version

### Module(s)

mini.completion

### Description

mini.completion crashes when trying to complete inside open open brackets in Ada.

The LSP server is ada_language_server of version 24.0.4, configured with `require'lspconfig'.als.setup {}` and I have jiangmiao/auto-pairs installed

### Neovim version

v0.9.5

### Steps to reproduce

1. Open an Ada file
2. Write a function name (in my case it was `Put_Line`)
3. Open a bracket (`(`)
4. Get error described in Actual behavior

### Actual behavior

Gives this error:
```
Error executing vim.schedule lua callback: /usr/share/nvim/runtime/lua/vim/lsp/util.lua:1407: 'replacemen
t string' item contains newlines                                                                         
stack traceback:                                                                                         
        [C]: in function 'nvim_buf_set_lines'                                                            
        /usr/share/nvim/runtime/lua/vim/lsp/util.lua:1407: in function 'stylize_markdown'                
        .../share/nvim/lazy/mini.completion/lua/mini/completion.lua:1085: in function 'show_signature_win
dow'                                                                                                     
        .../share/nvim/lazy/mini.completion/lua/mini/completion.lua:1056: in function 'callback'         
        /usr/share/nvim/runtime/lua/vim/lsp.lua:2020: in function 'handler'                              
        /usr/share/nvim/runtime/lua/vim/lsp.lua:1393: in function ''                                     
        vim/_editor.lua: in function <vim/_editor.lua:0>                                                 
Press ENTER or type command to continue
```

**Comment by echasnovski** [comment]
Thanks for the issue!

This seems to happen because 'mini.completion' tries to show signature information and fails due to an upstream error.

I don't have any LSP server for Ada files installed (and would like to keep it that way if possible), so would you mind getting some debug information to help solve this issue?

Here are the steps:
- Execute the following (very long, but acceptable) command `:lua _G.log, orig = {}, vim.lsp.util.stylize_markdown; vim.lsp.util.stylize_markdown = function(...) table.insert(_G.log, {...}); return orig(...) end` .  This will wrap offending core function to track how it was called.
- Reproduce the issue (preferably so that no information or signature help windows is shown prior to it).
- Execute the following command `:lua vim.fn.append(0, vim.split(vim.inspect(_G.log), '\n'))`. This will put some new lines at the top of the current buffer.
- Copy new lines and paste them here.

**Comment by Orest58008** [comment]
```lua
{ { 4, { "```ada", "procedure Put_Line\n  (Item : String)", "```" }, {
      wrap_at = 80
    } } }
```

**Comment by echasnovski** [comment]
Thanks for the follow up!

This is indeed an upstream Neovim issue which got resolved in (hopefully soon to be released) 0.10 version. However, I did an update which should make this type of signature lines work on Neovim<0.10.

**Comment by Orest58008** [comment]
Thank you for your time, the commit solved the issue!
